TACC-Cloud/hazmapper
# Task/WG-368 improve unit testing of requests

## Overview: ##
 improve unit testing of requests
 
 * use `msw`
 * ensure we fail if any non-mocked requests are made
 * hide some warnings that we aren't interested in


## PR Status: ##

## Related Jira tickets: ##

* [WG-368](https://tacc-main.atlassian.net/browse/WG-368)

## Summary of Changes: ##

## Testing Steps: ##
1. Run unit tests
2. Check output and ensure no warnings (except for default param warnings related to core-components)


## Notes: ##
- [x] refactor to match typical msw usage
